e2e/errors.go

```diff
@@ -41,6 +41,11 @@ func isRetryableAPIError(err error) bool {
 		return true
 	}
 
+	// "transport is closing" is an internal gRPC err, we can not use ErrConnClosing
+	if strings.Contains(err.Error(), "transport is closing") {
+		return true
+	}
```

Comment:
instead can we simply:
```
    return strings.Contains(err.Error(), "transport is closing")
```

Reply:
that is not how the other checks do it either, I prefer to stick with how the rest of the function works